File: send_message.py
import watson_developer_cloud

from decouple import config

# Set up Assistant service.
service = watson_developer_cloud.AssistantV1(
	username=config("WATSON_USERNAME"),  # replace with service username
	password=config("WATSON_PASSWORD"),  # replace with service password
	version=config("WATSON_VERSION")
)
workspace_id = config("WORKSPACE_ID")  # replace with workspace ID

# Start conversation with empty message.
response = service.message(
	workspace_id=workspace_id,
	input={
		'text': ''
	}
)

# Print the output from dialog, if any.
if response['output']['text']:
	print(response['output']['text'][0])

File: sendMessage.js
var AssistantV1 = require("watson-developer-cloud/assistant/v1");
require('dotenv').config()

// Setup assistant service wrapper
var service = new AssistantV1({
	username: process.env.WATSON_USERNAME,
	password: process.env.WATSON_PASSWORD,
	version: process.env.WATSON_VERSION
})

var workspaceId = process.env.WORKSPACE_ID;

service.message({
	workspace_id: workspaceId
}, processResponse);

function processResponse(err, response) {
	if (err) {
		// in case something went wrong.
		console.error(err);
		return;
	}

	// Display the output from dialog, if any
	if (response.output.text.length != 0) {
		console.log(response.output.text[0]);
	}
}
